app/routes/downloadPaymentsByYearSummary.ts

```diff
@@ -0,0 +1,24 @@
+import { Request, ResponseToolkit, ResponseObject } from '@hapi/hapi';
+import config from '../config';
+import * as utils from '../utils';
+
+const urlcsv = `${config.backendEndpoint}` + '/downloadPaymentsByYearSummary';
+
+module.exports = {
+  method: 'GET',
+  path: '/downloadPaymentsByYearSummary',
+  handler: async (_request: Request, _response: ResponseToolkit) => {
+    try {
+      const content = await utils.getBuffer(urlcsv);
+      return _response
+        .response(content)
+        .type('application/csv')
+        .header(
+          'Content-Disposition',
+          'attachment; filename="ffc-year-payments-summary.csv"'
```

Comment:
can you change the filename to ffc-payments-by-year.csv please? This is so it sounds similar to the this api route as well as the FE page.

Reply:
Thanks, I fixed it.